src/legacy/core_plugins/metrics/public/visualizations/components/_top_n.scss

```diff
@@ -19,11 +29,17 @@
   color: $tvbValueColor;
   padding: $euiSizeXS;
   padding-bottom: 0;
+  width: 3%;
```

Comment:
Why was this added? It should just take the space it needs IMHO - tested with a few different values and it looks fine for me.

Reply:
Removed